### PR TITLE
Validate paymenr request status

### PR DESCRIPTION
**Pull request by nnorbert**
Validate paymenr request status

### CODE REVIEW

File: apps/payments-api/routes/schemas/validations/validators/paymentRequestStatus.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { ErrorTypes, ValidatorFn } from ".";
+
+const validate: ValidatorFn = (value, field, rootData, errors) => {
+  console.log(rootData);
```
Comment (alfonsograziano) [inline apps/payments-api/routes/schemas/validations/validators/paymentRequestStatus.ts:4]: Please remove this cl

File: apps/payments-api/routes/schemas/validations/validators/paymentRequestStatus.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  const inValid = value === "active" && rootData.providers?.length === 0;
```
Comment (alfonsograziano) [inline apps/payments-api/routes/schemas/validations/validators/paymentRequestStatus.ts:5]: Why do we have a capital `V`?

**Review by alfonsograziano — APPROVED** [review]
LGTM overall :)

File: apps/payments-api/routes/schemas/validations/validators/paymentRequestStatus.ts (hunk 1 above)
Comment (nnorbert) [reply apps/payments-api/routes/schemas/validations/validators/paymentRequestStatus.ts]: Done

File: apps/payments-api/routes/schemas/validations/validators/paymentRequestStatus.ts (hunk 2 above)
Comment (nnorbert) [reply apps/payments-api/routes/schemas/validations/validators/paymentRequestStatus.ts]: I can't really remember, but it is just a mistake. Fixed.

**Pull request merged**